package.json

```diff
@@ -26,6 +26,7 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@guardian/node-riffraff-artifact": "^0.3.2",
```

Comment:
This was defined in two of the workspace package.json files, but should be at the top level.
I've also bumped the version, which is compatible with github actions (specifically the `GITHUB_RUN_NUMBER` env var)